pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
```

Comment:
I agree, this doesn't make sense to me either. It's needed only because setuptools has a bug that without it `pip install -e . ` won't include `third_party` folder, which I also noted in the PR description. I added a comment here for easier future reference.